Source/WebKit/NetworkProcess/Classifier/ResourceLoadStatisticsStore.cpp

```diff
@@ -86,7 +86,7 @@ constexpr auto countSubresourceUniqueRedirectsToQuery = "SELECT COUNT(*) FROM Su
 // INSERT OR IGNORE Queries
 constexpr auto insertObservedDomainQuery = "INSERT INTO ObservedDomains (registrableDomain, lastSeen, hadUserInteraction,"
     "mostRecentUserInteractionTime, grandfathered, isPrevalent, isVeryPrevalent, dataRecordsRemoved, timesAccessedAsFirstPartyDueToUserInteraction,"
-    "timesAccessedAsFirstPartyDueToStorageAccessAPI, isScheduledForAllButCookieDataRemoval, mostRecentWebPushInteractionTime) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)"_s;
+    "timesAccessedAsFirstPartyDueToStorageAccessAPI, isScheduledForAllButCookieDataRemoval, mostRecentWebPushInteractionTime, seenOutsideEnhancedSecurity) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)"_s;
```

Comment:
I agree this is a reasonable change in theory, but it's a little more complicated. ResourceLoadStatistics is tightly coupled with cross-site tracking prevention (i.e., ITP) such that we don't have this database if a customer disables tracking prevention. We can think about decoupling these and actually tracking some of these statistics separately from the tracking prevention, but that's a larger project.